src/pages/faq/cors.md

```diff
@@ -0,0 +1,250 @@
+| **Access-Control-Allow-Headers** | `headers`         | Used in response to a preflight request to indicate which headers can be used when making the actual request, aside from the <a href="#simple-requests">simple headers</a>, which are always allowed. |
+| Access-Control-Allow-Credentials | `true` or `false` | Whether or not the request can be made with credentials.                                           |
+| Access-Control-Expose-Headers    | `headers`         | Whitelists the headers that the browser is allowed to access.                                      |
+| Access-Control-Max-Age           | `seconds`         | Indicates how long the results of a preflight request can be cached.                               |
+
+### Browser Headers (Request)
+
+The browser automatically sends the appropriate headers for CORS in every request to the server, including the preflight requests. Please note that the headers below are for reference only, and **should not be set in your app code** (the browser will ignore them).
+
+#### All Requests
+
+| Header     | Value    | Description                          |
+|------------|----------|--------------------------------------|
+| **Origin** | `origin` | Indicates the origin of the request. |
+
+#### Preflight Requests
+
+| Header                            | Value     | Description                                                                                       |
+|-----------------------------------|-----------|---------------------------------------------------------------------------------------------------|
+| **Access-Control-Request-Method** | `method`  | Used to let the server know what method will be used when the actual request is made.             |
+| Access-Control-Request-Headers    | `headers` | Used to let the server know what non-simple headers will be used when the actual request is made. |
+
+
+## Solutions for CORS Errors
+
+### A. Enabling CORS in a server you control
+
+The correct and easiest solution is to enable CORS by returning the <a href="#server-headers-response-">right response headers</a> from the web server or backend and responding to preflight requests, as it allows to keep using `XMLHttpRequest`, `fetch`, or abstractions like `HttpClient` in Angular.
+
+These are the recommended `Access-Control-Allow-Origin` headers for Ionic apps. Please note that these headers have to be sent from the server, and don't belong in your app code.
+
+**Capacitor**
+
+```http
+Access-Control-Allow-Origin: capacitor://localhost, http://localhost
+```
+
+*Replace `localhost` with your own hostname if you have changed the default in the Capacitor config.*
+
+**Ionic WebView plugin on Cordova**
+
+```http
+Access-Control-Allow-Origin: http://localhost:8080
+```
+
+*Replace port `8080` with your own if you have changed the default in the plugin config.*
```

Comment:
As commented before, WebView 3.x doesn't have a port anymore. 
As we are still supporting WebView 2.x, it's probably better to have two sections, one for 3.x with the `ionic://localhost`for iOS and `http://localhost` for android and other for 2.x with the port.
Well, in fact, even in 2.x it can use `ionic://localhost` for iOS 11+ if you use `UseScheme` preference.

Reply:
ionic is on webview plugin, capacitor on capacitor

Reply:
I fixed all the origins, moved them to different tables depending on the webview and recommend to reflect the origin instead, since only one can be specified in the `Access-Control-Allow-Origin` header. Thanks!